core/src/main/java/io/kestra/core/runners/Executor.java

```diff
@@ -34,6 +35,12 @@ public class Executor {
     private ExecutionResumed executionResumed;
     private ExecutionResumed joinedExecutionResumed;
 
+    /**
+     * List of {@link ExecutionKilled} to be propagated part of the execution.
+     */
+    @JsonIgnore
+    private List<ExecutionKilled> executionKilledEvents;
```

Comment:
```suggestion
    private List<ExecutionKilled> executionsKilled;
```

We don't use the suffix events anywhere so better to be consistent.
Also the JsonIgnore didn't seems to be useful here, let's KISS

Reply:
Regarding the `JsonIgnore` I'm not sure how the `Executor` object may be used in the OSS part. If the executor is serialized somewhere, I don't want the events to be included. Currently, it seems that the `Executor#serialize` is not used (at-least in Kestra OSS°.

Reply:
The executor is only serialized in the Kafka implementation as it is stored in a statestore and we do want to have all its components saved to be able to have safepoints during the Stream.
Moreover, this is more like a KISS principle, or just for consistency share